shuup/importer/transforms.py

```diff
@@ -125,7 +127,16 @@ def py2_read_file(data, filename):
 def py3_read_file(data, filename):
     got_data = set()
     data = []
-    with open(filename, encoding="utf-8") as f:
+
+    bytes = min(32, os.path.getsize(filename))
+    raw = open(filename, 'rb').read(bytes)
```

Comment:
You should also close the file. Also.. it doesn't matter if the file is shorter than the bytes read, so there is no need to do the getsize. I suggest simplying lines 131-137 to:
```python
with open(filename, 'rb') as fp:
    encoding = 'utf-8-sig' if fp.read(2) == codecs.BOM_UTF8 else 'utf-8'
```